contracts/src/starknet/rewaave/tokens/ETHstaticAToken.cairo

```diff
@@ -18,13 +18,24 @@ from rewaave.tokens.claimable import (
     claimable_claim_rewards, claimable_push_acc_rewards_per_token, claimable_before_token_transfer,
     claimable_get_acc_rewards_per_token)
 
+@contract_interface
+namespace ITokenBridge:
+    func mint_rewards(recipient : felt, amount : Uint256):
+    end
+end
+
+@storage_var
+func l2_token_bridge() -> (address : felt):
+end
+
 @constructor
 func constructor{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
         name : felt, symbol : felt, decimals : felt, initial_supply : Uint256, recipient : felt,
-        controller : felt):
+        controller : felt, tokens_bridge : felt):
```

Comment:
typo: `token_bridge` instead of `tokens_bridge`. also, we should probably name it `l2_token_bridge` to be explicit.